homeassistant/components/netgear/router.py

```diff
@@ -210,6 +210,12 @@ async def async_update_device_trackers(self, now=None) -> bool:
 
         for device in self.devices.values():
             device["active"] = now - device["last_seen"] <= self._consider_home
+            if not device["active"]:
+                device["link_rate"] = None
+                device["signal"] = None
+                device["ip"] = None
+                device["ssid"] = None
+                device["conn_ap_mac"] = None
```

Comment:
If a device is not active anymore, wouldn't it make more sense to make the entity `unavailable`? We do this by returning `False` for the `available` property of the entity. WDYT? Setting them to `None` sets them to `unknown` and I don't think that is correct

Reply:
It's a device tracker, so for instance a phone is expected to disconnect when it leaves the Home.

But I don't know what would be more fitting: Unknown or Unavailable..

Reply:
To me personally `unavailable` implies an error whereas in this case it is expected for these values to be `unknown` if the device is `away`, i.e., not connected to the network.

The best would actually be to show `none` IMO but I AFAIK that's not possible currently without setting it as string value which makes it indistinguishable from an SSID with the same name.

Reply:
we already have `unknown` and `unavailable` as values meaning "nothing" so I don't want to add another one :)

> implies an error

In a way it is an error, because we can't fetch data for it because the device is not there. Unknown always feels like, we could fetch data from it, but we simply do not know (the device did not reply with that value)

Reply:
Updated the code to mark the entity as unavailable instead